org.eclipse.gef.examples.shapes/src/org/eclipse/gef/examples/shapes/model/Shape.java

```diff
@@ -90,8 +90,12 @@ public abstract class Shape extends ModelElement {
 	 * @see #setPropertyValue(Object, Object)
 	 */
 	static {
-		descriptors = new IPropertyDescriptor[] { new TextPropertyDescriptor(XPOS_PROP, ShapesExampleMessages.Shape_X), // id and description pair
-				new TextPropertyDescriptor(YPOS_PROP, ShapesExampleMessages.Shape_Y), new TextPropertyDescriptor(WIDTH_PROP, ShapesExampleMessages.Shape_Width),
+		descriptors = new IPropertyDescriptor[] { new TextPropertyDescriptor(XPOS_PROP, ShapesExampleMessages.Shape_X), // id
+																														// and
+																														// description
+																														// pair
+				new TextPropertyDescriptor(YPOS_PROP, ShapesExampleMessages.Shape_Y),
```

Comment:
Just to fix the broken comment:
```suggestion
		// id and description pair
		descriptors = new IPropertyDescriptor[] { new TextPropertyDescriptor(XPOS_PROP, ShapesExampleMessages.Shape_X),
				new TextPropertyDescriptor(YPOS_PROP, ShapesExampleMessages.Shape_Y),
				new TextPropertyDescriptor(WIDTH_PROP, ShapesExampleMessages.Shape_Width),
				new TextPropertyDescriptor(HEIGHT_PROP, ShapesExampleMessages.Shape_Height), };
```